src/components/metrics/CountCard.jsx

```diff
@@ -1,20 +1,34 @@
 import React from "react"
 import PropTypes from "prop-types"
+import _ from "lodash"
 
-const CountCard = ({ title, count }) => (
+const CountCard = ({ title, count = null, footer = null }) => (
```

Comment:
Made the count card more flexible by allowing a footer to be supplied. For the new cards, this contains the inputs.